samples/zigbee/shell/README.rst

```diff
@@ -0,0 +1,153 @@
+.. _zigbee_shell_sample:
+
+Zigbee: Shell
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This :ref:`Zigbee <ug_zigbee>` shell sample demonstrates a Zigbee router (with the possibility of being a coordinator) that uses the :ref:<lib_zigbee_shell> library for interaction.
```

Comment:
Link to `:ref:<lib_zigbee_shell>` is broken in documentation

Reply:
Fixed!